packages/components/src/components/hds/stepper/nav/step.hbs

```diff
@@ -0,0 +1,87 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+{{! template-lint-disable require-context-role no-invalid-role }}
+{{! template-lint-disable require-presentational-children }}
+{{#if this.isNavInteractive}}
+  <li class={{this.classNames}} ...attributes role={{if this.isNavInteractive "presentation"}}>
+    <button
+      class="hds-stepper-nav__step-content hds-stepper-nav__step-button"
+      id={{this._stepId}}
+      tabindex={{unless (eq this.status "active") "-1"}}
+      type="button"
+      role="tab"
+      aria-controls={{this.coupledPanelId}}
+      aria-selected={{if (eq this.status "active") "true" "false"}}
+      aria-disabled={{if (eq this.status "incomplete") "true" "false"}}
+      {{on "click" this.onStepChange}}
+      {{on "keyup" this.onKeyUp}}
+      {{this._setUpStep this.didInsertNode this.willDestroyNode}}
+    >
+      <div class="hds-stepper-nav__step-progress">
+        <Hds::Stepper::Step::Indicator
+          @text="{{this.stepNumber}}"
+          @status={{this.stepIndicatorStatus}}
+          @isInteractive={{true}}
+          class="hds-stepper-nav__step-indicator"
+        />
+      </div>
+      <div class="hds-stepper-nav__step-text">
+        <Hds::Text::Body class="hds-stepper-nav__step-title" @tag={{this.titleTag}} @size="200" @weight="semibold">
+          {{yield to="title"}}
+          <span class="sr-only">{{this.statusSrOnlyText}}</span>
+        </Hds::Text::Body>
+        <Hds::Text::Body
+          class="hds-stepper-nav__step-description"
+          @tag="div"
+          @size="100"
+          @weight="regular"
+          @color="regular"
```

Comment:
`regular` isn't one of the possible values for color:  
 https://helios.hashicorp.design/components/text#textbody